packages/replay-next/src/suspense/SearchCache.ts

```diff
@@ -69,18 +71,34 @@ export function assertSourceSearchResultMatch(
 }
 
 export const searchCache = createStreamingCache<
-  [replayClient: ReplayClientInterface, query: string, includeNodeModules: boolean, limit?: number],
+  [
+    replayClient: ReplayClientInterface,
+    query: string,
+    defaultFilter: boolean,
```

Comment:
I _think_ "default filter" means hiding node_modules and CDN files?

I see VS Code has a setting that this seems to be based off of "use exclude settings and ignored filed" but I think ours is a bit different. VS Code's seems to be based off of things like the `.gitignore` file, where as ours is just hard-coded to filter out NPM packages.

So can we name this something more semantically meaningful than "default"? :)